src/lib/support/CHIPArgParser.cpp

```diff
@@ -43,6 +43,7 @@
 #include <stdarg.h>
 #include <stdint.h>
 #include <string.h>
+#include <strings.h>
```

Comment:
Actually, this file stopped compiling with new Zephyr because the code in chip-lib.cmake for filtering out "-std=c++11" from Zephyr compilation flags is no longer valid after Zephyr has switched to using CMake generator expressions in a dependent component. We can talk about details, but the bottom line is that you don't need to fix this file, you can just add "-std=gnu++11" at the end of CXX flags passed to GN to override that coming from Zephyr.

Reply:
Thanks, it's great that you found a problem root cause! It's done, as you suggested.